packages/fluentui/react-northstar/test/specs/components/Chat/Chat-test.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+
+import { handlesAccessibility, isConformant, getRenderedAttribute } from 'test/specs/commonTests';
+import { mountWithProvider } from 'test/utils';
+
+import { Chat } from 'src/components/Chat/Chat';
+import { implementsCollectionShorthandProp } from '../../commonTests/implementsCollectionShorthandProp';
+import { ChatItem } from 'src/components/Chat/ChatItem';
+
+const chatImplementsCollectionShorthandProp = implementsCollectionShorthandProp(Chat);
+
+describe('Chat', () => {
+  isConformant(Chat, {
+    testPath: __filename,
+    constructorName: 'Chat',
+  });
+  chatImplementsCollectionShorthandProp('items', ChatItem, { mapsValueToProp: 'message' });
+
+  describe('accessibility', () => {
+    handlesAccessibility(Chat);
+  });
+
+  describe('child behaviors', () => {
+    const items = [
+      {
+        message: <Chat.Message content="Hello" author="Robert Tolbert" timestamp="10:15 PM" mine />,
+      },
+    ];
+
+    it('uses default when not overriden', () => {
+      const chatMessage = mountWithProvider(<Chat items={items} />).find('ChatMessage');
+      expect(getRenderedAttribute(chatMessage, 'role', '')).toBe(undefined);
+    });
+
+    it('uses child behavior for message', () => {
+      const messageBehavior = () => ({
+        attributes: {
+          root: {
+            role: 'button',
```

Comment:
should fix your type problem

```suggestion
            role: 'button' as const,
```

Reply:
I spent like one hour trying to figure out why types dont work. thank you :)